scheds/rust/scx_lavd/src/bpf/main.bpf.c

```diff
@@ -1919,30 +2187,118 @@ void BPF_STRUCT_OPS(lavd_enqueue, struct task_struct *p, u64 enq_flags)
 	put_global_rq(p, taskc, cpuc, enq_flags);
 }
 
+static bool is_kernel_task(struct task_struct *p)
+{
+	return p->mm == NULL;
```

Comment:
Testing `PF_KTHREAD` is probably more idiomatic.

Reply:
Fixed. Thanks!